grpc/client.go

```diff
@@ -39,7 +40,26 @@ func ClientSetup(c *cmd.GRPCClientConfig, stats metrics.Scope) (*grpc.ClientConn
 		return nil, err
 	}
 	ci := clientInterceptor{stats.NewScope("gRPCClient"), clock.Default()}
-	creds := bcreds.NewClientCredentials(rootCAs, []tls.Certificate{clientCert})
+
+	//TODO(@cpu): It's janky to only look at the first server address here. Do we
+	//need to restore the `clientTransportCredentials`?
```

Comment:
Yes - I was hoping someone might see a way to adapt this so that it works without restoring the `clientTransportCredentials` but I haven't seen any alternatives posed so I think its probably best to just restore our impl.